### ISSUE TITLE
Chrome 94 breaks this theme

### ISSUE THREAD

**Issue by transparentech** [rbika/easy-purple-incognito-theme#2]
Hey, I love this theme and have even customized it for myself (https://github.com/transparentech/easy-purple-incognito-theme) but it looks like Chrome 94 broke it. Any idea what was changed and how we can fix it?

**Comment by rbika** [comment]
Hi, I'm running Chrome Version 94.0.4606.61 on Windows 10 and it's working as expected. Whats OS / chrome version are you using, and what you mean by "broke it"? Could you be more specific please?

**Comment by transparentech** [comment]
By "broke it", I mean that incognito windows are not showing in purple. I'm on a Mac with Chrome version 94.0.4606.71 (Official Build) (x86_64). Here's a screenshot:

<img width="1504" alt="incognito-broken" src="https://user-images.githubusercontent.com/334795/135728719-22341238-95c0-4b72-a58a-17dd14f6cfd5.png">

**Comment by rbika** [comment]
I see, @transparentech. I'll take a look. Feel free to open a PR if you find out what's the problem 😉

**Comment by rbika** [comment]
I was able to test it on a M1 Mac machine and it is also working without problems. Could you try to uninstall and reinstall the version from [chrome store](https://chrome.google.com/webstore/detail/easy-purple-incognito-mod/jgolkknoinekgblmggbdkdbempdhabpj?hl=en), please? 

I also made an update and applyed some changes you made in your forked version 🙂

**Comment by transparentech** [comment]
Hmm, resetting to the default theme and then re-installing didn't help, and removing the Extension configs and re-installing didn't either. So I wiped the entire Chrome config and after that I was able to install the theme and it worked. I would guess that my other extension development must have broken it somehow.

In any case, sorry about the noise and thanks for incorporating some of my changes!
